src/components/CodeInput/styled/index.js

```diff
@@ -60,26 +69,6 @@ const StyledInput = attachThemeAttrs(styled.input)`
         background-color: ${props.palette.background.main};
         border: 2px solid ${props.palette.error.main};
         background-clip: padding-box;
-
-        :focus,
-        :active {
-            background-color: ${props.palette.background.main};
-            border: 2px solid ${props.palette.error.main};
-            box-shadow: ${props.shadows.error};
```

Comment:
here you can not remove the shadow when doing focus

Reply:
sorry, my bad... now is returned that specific block, the "box-shadow" along with a few properties that were already there.